v2/bigtable-changestreams-to-hbase/README.md

```diff
@@ -8,19 +8,53 @@ The [BigtableToHBase](src/main/java/com/google/cloud/teleport/v2/templates/Bigta
 * Bigtable table with change streams enabled
 * Hbase instance that is accessible from Dataflow
 
-#### Beware
+#### Caveat - provision Hbase resources to handle Dataflow traffic
 
 * Dataflow when strongly provisioned can translate Bigtable input QPS directly to Hbase, the user should configure Hbase so that Hbase can handle that QPS without crashing.
```

Comment:
Can you find a better word than "Strongly provisioned"?

Reply:
Changed wording pls review again